stl/inc/xutility

```diff
@@ -541,7 +541,8 @@ struct less_equal<void> {
 template <class _Fx>
 struct _Ref_fn { // pass function object by value as a reference
     template <class... _Args>
-    constexpr decltype(auto) operator()(_Args&&... _Vals) { // forward function call operator
+    constexpr decltype(auto) operator()(_Args&&... _Vals) noexcept(
+        noexcept(_STD invoke(_Fn, _STD forward<_Args>(_Vals)...))) { // forward function call operator
         if constexpr (is_member_pointer_v<_Fx>) {

```

Comment:
The two branches are both equivalent to `_STD invoke(_Fn, _STD forward<_Args>(_Vals)...);` can we remove the branch here?

Reply:
> We can use _Select_invoke_traits

Hmm, sorry, I pushed too quickly